unitex/src/fr/umlv/unitex/leximir/delas/EditorDelas.java

```diff
@@ -0,0 +1,1364 @@
+        this.getjTable1().setRowSorter(rowSorter);
+        this.getjTable1().removeAll();
+        if (text.length() == 0) {
+            rowSorter.setRowFilter(null);
+        } else {
+            rowSorter.setRowFilter(RowFilter.regexFilter(text));
+        }
+        jTable1.setModel(rowSorter.getModel());
+        jLabel13.setText(String.valueOf(this.getjTable1().getRowCount()));
+
+    }
+
+    private void jMenuStatisticsActionPerformed(java.awt.event.ActionEvent evt) {
+
+    }
+
+    private void jMenuNewMouseClicked(java.awt.event.MouseEvent evt) {
+
+    }
+
+    private void jMenuStatisticsMouseClicked(java.awt.event.MouseEvent evt) {
+
+        Map<String, HashMap<String, String>> dic_POS_stat = new HashMap<>();
+        for (int i = 0; i < this.getjTable1().getRowCount(); i++) {
+            String dic = (String) this.getjTable1().getValueAt(i, 7);
+            String value = (String) this.getjTable1().getValueAt(i, 0);
+            if (!dic_POS_stat.containsKey(dic)) {
+                dic_POS_stat.put(dic, new HashMap<String, String>());
+                dic_POS_stat.get(dic).put(value, "1");
+            } else {
+                if (dic_POS_stat.get(dic).containsKey(value)) {
+                    int count = Integer.parseInt(dic_POS_stat.get(dic).get(value)) + 1;
+                    //dic_POS_stat.get(dic).replace(value, String.valueOf(count));
+                    if (dic_POS_stat.get(dic).containsKey(value)) {
+                        dic_POS_stat.get(dic).put(value, String.valueOf(count));
+                    }
+                } else {
+                    dic_POS_stat.get(dic).put(value, "1");
+                }
+            }
+        }
+
+        Map<String, String> pOS_stat = new HashMap<>();
+        for (int i = 0; i < this.getjTable1().getRowCount(); i++) {
+            String value = (String) this.getjTable1().getValueAt(i, 0);
+            if (!pOS_stat.containsKey(value)) {
+                pOS_stat.put(value, "1");
+            } else {
+                int count = Integer.parseInt(pOS_stat.get(value)) + 1;
+                //pOS_stat.replace(value, String.valueOf(count));
+                if (pOS_stat.containsKey(value)) {
+                    pOS_stat.put(value, String.valueOf(count));
+                }
+
+            }
+        }
+
+        List<Object[]> dicPos = new ArrayList<Object[]>();
+        dicPos.add(new Object[]{"Dic", "POS", "Number"});
+        for (Map.Entry<String, HashMap<String, String>> f : dic_POS_stat.entrySet()) {
+            String key = f.getKey();
+            for (Map.Entry<String, String> p : f.getValue().entrySet()) {
+                dicPos.add(new Object[]{key, p.getKey(), p.getValue()});
+
+            }
+        }
+        GlobalProjectManager.search(null).getFrameManagerAs(UnitexInternalFrameManager.class)
+                .newStatisticOutput(dicPos);
+
+    }
+
+    private void jMenuDeleteMouseClicked(java.awt.event.MouseEvent evt) {
```

Comment:
Could the following methods : `jMenuDeleteMouseClicked`, `jMenuItemInsertAfterActionPerformed`, `jMenuItemInsertBeforeActionPerformed`, `jMenuViewMouseClicked`, `jMenuEditMouseClicked`, `jMenuBeforeMouseClicked` and `jMenuAfterMouseClicked`
be refactored to reduce code duplication ?

Reply:
Done !